yare-model/src/main/java/com/sabre/oss/yare/core/model/Attribute.java

```diff
@@ -27,7 +27,7 @@
 import java.lang.reflect.Type;
 import java.util.Objects;
 
-public class Attribute {
+public final class Attribute {
```

Comment:
import jdk.nashorn.internal.ir.annotations.Immutable;


We can add these annotations to model classes, making it more obvious that they are immutable

Reply:
In my opinion we don't need such annotations. It will serve as meaningless comment, without any validation or metadata.

Reply:
Agree with @sobolewskikamil . Actually we cannot even assure immutability (Attribute's value is Object provided by client - if it is mutable then Attribute is mutable also).

Reply:
Yeah, I was wondering whether we should, for example, return a deep copy of a given value in getValue, but that would add a lot of overhead to a simple operation, as we would need to use reflection

Reply:
+1